gammapy/maps/axes.py

```diff
@@ -2973,6 +2973,59 @@ def to_header(self, format="gadf", idx=0):
 
         return header
 
+    def group_table(self, interval_edges):
```

Comment:
Can't you re-use the code in `GTI.group_table`? Is it possible to avoid the code duplication here?

Reply:
The initial idea was to wrap `GTI.group_table`, but unfortunately the method does not handle bin indexes. This would result in either having to use `coord_to_idx`, defeating the point of the change, or writing an equivalent translator code that would probably require more space.